core/src/main/java/com/googlecode/psiprobe/beans/stats/providers/MultipleSeriesProvider.java

```diff
@@ -36,7 +36,7 @@
 public class MultipleSeriesProvider extends AbstractSeriesProvider {
   
   /** The stat name prefix. */
-  private String statNamePrefix;
```

Comment:
Ok, makes sense. The inner class is private too. No harm in maintaining privacy, but like I said, Eclipse is very opinionated.

I'm fine with inner classes that encapsulate something very specific, but I can understand the desire to move it out to maybe a package-private class for testing purposes. Still, the tests on a containing class *should* exercise its inner classes completely.